i18n/en.json

```diff
@@ -3827,8 +3819,20 @@
     "translation": "Unable to rename the team, the name is already in use."
   },
   {
-    "id": "app.user.complete_switch_with_oauth.blank_email.app_error",
-    "translation": "Unable to complete SAML login with an empty email address."
+    "id": "app.terms_of_service.create.app_error",
```

Comment:
Because you have autogenerated and you don't have the enterprise repository you removed a lot of valid enterprise strings. You have to edit it by hand, or let us regenerate it for you.

Reply:
I'm wondering whether this should be considered a bug ? This essentially means a contributor cannot run `i18n-extract`. We should fix that to selectively check either just TE repo or both TE and EE.

Reply:
Yes, is a bug in the design, probably we can just ensure that an `ent` prefix is used for enterprise strings (with the same i18n-extract tool) and then, make the TE generation aware of no `ent` strings must be altered.

Reply:
Created https://mattermost.atlassian.net/browse/MM-25619